### PR TITLE
AC-363: Unit test - PatientDashboardVitalsPresenter

### PR DESCRIPTION
**Pull request by shivtej1505**

### CODE REVIEW

File: openmrs-client/src/test/java/org/openmrs/mobile/test/presenters/PatientDashboardVitalsPresenterTest.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+import rx.Observable;
+
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(PowerMockRunner.class)
```
Comment (AdamGrzybkowski) [inline openmrs-client/src/test/java/org/openmrs/mobile/test/presenters/PatientDashboardVitalsPresenterTest.java]: @shivtej1505 We can remove this line after last commit. It is now by default in ACUnitTestBase

Comment (AdamGrzybkowski) [inline openmrs-client/src/test/java/org/openmrs/mobile/test/presenters/PatientDashboardVitalsPresenterTest.java]: Feel free to merge this once the above is done :) Good job : )

**Comment by shivtej1505** [comment]
@AdamGrzybkowski , thanks :) Merging.

**Comment by shivtej1505** [comment]
I take this as an opportunity to remove all unused imports :)

**Comment by AvijitGhosh82** [comment]
@shivtej1505 , Why did you merg a PR in which the Codacy test failed? This is bad practice.

**Comment by shivtej1505** [comment]
@AvijitGhosh82 , sorry for that. I must've merged it in hurry. BTW, there were many unused imports in the codebase and Codacy didn't detect them so I removed them altogether in this [PR](https://github.com/openmrs/openmrs-contrib-android-client/pull/352) including usused imports in this file .
Thanks for the feedback :)

**Pull request merged**